modules/cnf-topology-aware-lifecycle-operator-installation-cli.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="installing-topology-aware-lifecycle-operator-using-cli_{context}"]
+= Installing the Topology-Aware Lifecycle Operator by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the Topology-Aware Lifecycle Operator.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `Namespace` custom resource (CR) for the Topology-Aware Lifecycle Operator:
+.. Define the `Namespace` CR and save the YAML file, for example, `talo-namespace.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-operators
```

Comment:
```suggestion
  name: openshift-cluster-group-upgrades
```

Reply:
Sure! I wanted to have it as a placeholder and we can edit it when we know more